README.md

```diff
@@ -10,3 +22,371 @@ By using Tidoop you will not need to copy data from remote to HDFS anymore, but
+log4j:WARN No appenders could be found for logger (com.telefonica.iot.tidoop.http.HttpClientFactory).
+log4j:WARN Please initialize the log4j system properly.
+log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
+Testing CKANBackend.getPackages
+Testing CKANBackend.getResources
+Testing CKANBackend.getNumRecords
+Testing CKANBackend.getRecords
+Testing CKANBackend.doCKANRequest (no payload)
+Testing CKANBackend.doCKANRequest (with payload)
+Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.209 sec
+Running com.telefonica.iot.tidoop.hadoop.ckan.CKANRecordReaderTest
+Testing CKANRecordReader.getCurrentValue
+Testing CKANRecordReader.close
+Testing CKANRecordReader.initialize
+Testing CKANRecordReader.getProgress
+Testing CKANRecordReader.nextKeyValue
+Testing CKANRecordReader.getCurrentKey
+Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.191 sec
+Running com.telefonica.iot.tidoop.hadoop.ckan.CKANInputFormatTest
+Testing CKANInputFormat.setCKANEnvironmnet
+Testing CKANInputFormat.addCKANInput
+Testing CKANInputFormat.getSplits (resource)
+Testing CKANInputFormat.getSplits (package)
+Testing CKANInputFormat.getSplits (organization)
+Testing CKANInputFormat.addCKANInput
+Testing CKANInputFormat.createRecordReader)
+Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.757 sec
+Running com.telefonica.iot.tidoop.hadoop.ckan.CKANInputSplitTest
+Testing CKANInputSplit.getResId
+Testing CKANInputSplit.readFields
+Testing CKANInputSplit.write
+Testing CKANInputSplit.getFirstRecordIndex
+Testing CKANInputSplit.getLocations
+Testing CKANInputSplit.getLength
+Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.047 sec
+
+Results :
+
+Tests run: 24, Failures: 0, Errors: 0, Skipped: 0
+
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 5.377 s
+[INFO] Finished at: 2015-02-26T13:00:31+01:00
+[INFO] Final Memory: 8M/104M
+[INFO] ------------------------------------------------------------------------
+```
+
+[Top](#section0)
+
+##<a name="section3"></a>Usage
+
+###<a name="section3.1"></a>CKAN extensions
+
+**Disclaimer:** If you are reading this section then you should be familiar with CKAN concepts and hierarchies. However, as a quick reminder, it will be said that CKAN organizes the data into <i>organizations</i>, having each organization a set of <i>packages</i> or <i>datasets</i>, having each package/organization a set of <i>resources</i>. Finally, each resource has a list of data records.
+
+A good way to learn about CKAN extensions is to have a look to the testing purpose MapReduce application distributed with Tidoop. This can be found at `src/main/java/com/telefonica/iot/tidoop/utils/CKANMapReduceExample.java`. This application is in charge of counting the number of bytes among all the configured innputs.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-tidoop/commit/7ed15d1641fa1714481bdba588c2c709a1956310